content/docs/serverless/serverless-driver.md

```diff
@@ -85,11 +86,15 @@ export default async function handler(
 }
 ```
 
+<Admonition type="note">
+The maximum response size for queries over HTTP is 10 MB as of version 0.6.0, increased from the 1 MB maximum supported in earlier versions of the driver.
+</Admonition>
+
 </CodeTabs>
 
-### Use experimental caching
+### Use experimental connection caching
 
-Connection caching allows Neon proxy to find the compute node attached to your database quicker. Connection caching is experimental and on opt-in only. You can try it by setting `fetchConnectionCache` to true in the `neonConfig` object.
+Connection caching allows the Neon proxy to quickly find the compute node attached to your database. Connection caching is experimental. You can enable it by setting `fetchConnectionCache` to `true` in the `neonConfig` object.
```

Comment:
I'm not sure either the old or new write-up is quite right here. Connection caching essentially provides server-side connection pooling, so that a new connection doesn't have to be set up for each query coming in over http.

(Also: connection caching is currently disabled, but will hopefully be re-enabled soon enough that we don't need to document this.)

Reply:
commenting out the connection caching section until we re-enable it

Reply:
I see we've addressed the issue, so I have added the section about connection caching back.